pom.xml

```diff
@@ -60,6 +60,9 @@
         <compiler.plugin.version>3.8.1</compiler.plugin.version>
         <exec.plugin.version>3.2.0</exec.plugin.version>
         <google.cloud.bom.version>26.33.0</google.cloud.bom.version>
+        <kafka.connect.plugin.version>0.11.1</kafka.connect.plugin.version>
```

Comment:
Don't think we need this, unless Aiven is publishing this connector to Confluent Hub (which considering the lineage of this project would be... interesting)